config/db.go

```diff
@@ -0,0 +1,71 @@
+package config
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/mattn/go-sqlite3"
+	"log"
+	"os"
+)
+
+var db *sql.DB
+
+func GetDB() *sql.DB {
+	return db
+}
+
+func InitializeDatabase() error {
+	// Получаем значение переменной окружения TODO_DBFILE
+	dbFile := os.Getenv("TODO_DBFILE")
+
+	// Проверяем, существует ли файл базы данных
+	_, err := os.Stat(dbFile)
+	install := false
+	if err != nil {
+		if os.IsNotExist(err) {
+			install = true
+		} else {
+			return fmt.Errorf("Файл базы данных не существует: %v", err)
+		}
+	}
+
+	// Подключаемся к базе данных
+	db, err = sql.Open("sqlite3", dbFile)
+	if err != nil {
+		return fmt.Errorf("Ошибка подключения: %v", err)
```

Comment:
Когда оборачиваем ошибку, то нужно использовать `%w` (wrap)
```suggestion
		return fmt.Errorf("Ошибка подключения: %w", err)
```